charts/pathfinder-rpc-server/README.md

```diff
@@ -2,35 +2,46 @@
 
 This chart includes the component in the table below in order to  isolate the pathfinder rpc server installation.
 
-For a detailed description please refer to [land-local](https://github.com/CirclesUBI/land-local) 
+For a detailed description please refer to [land-local](https://github.com/CirclesUBI/land-local)
  Repo                                                                     | Service               | Description                                                                              |
 |--------------------------------------------------------------------------|-----------------------|------------------------------------------------------------------------------------------|
 | [blockchain-indexer](https://github.com/CirclesUBI/blockchain-indexer)   | blockchain-indexer    | Indexes circles related on-chain events and writes them to a postgres db                 |
-| [pathfinder2](https://github.com/CirclesUBI/pathfinder2)                 | pathfinder2           | Finds transitive payment paths between untrusted users                                   | 
-| [pathfinder2-updater](https://github.com/CirclesUBI/pathfinder2-updater) | pathfinder2-updater   | Updates the 'pathfinder2' with data from the 'blockchain-indexer'                        | 
+| [pathfinder2](https://github.com/CirclesUBI/pathfinder2)                 | pathfinder2           | Finds transitive payment paths between untrusted users                                   |
+| [pathfinder2-updater](https://github.com/CirclesUBI/pathfinder2-updater) | pathfinder2-updater   | Updates the 'pathfinder2' with data from the 'blockchain-indexer'                        |
 | [pathfinder-proxy](https://github.com/CirclesUBI/pathfinder-proxy)       | pathfinder-proxy      | Maintains statistics, load balances and filters requests to the 'pathfinder2'            |
 
 
+#### Notes
+Since the installation of pathfinder and updater are in the same container, this charts needs to explicitely set the ports for both in the service so these can be located by other services such as the pathfinder-proxy on the network.
```

Comment:
```suggestion
Since the installation of pathfinder and updater are in the same container, this charts needs to explicitely set the ports for both in the service so these can be located by other services on the network, such as the pathfinder-proxy.
```